app/src/app/[lng]/[inventory]/page.tsx

```diff
@@ -119,6 +119,10 @@ export default function Home({ params: { lng } }: { params: { lng: string } }) {
     { skip: !inventory?.cityId || !inventory?.year },
   );
 
+  const { data: cityArea, isLoading: isCityAreaLoading } = api.useGetCityBoundaryQuery(inventory?.city.locode!, {
+    skip: !inventory?.city.locode,
+  });
+
```

Comment:
We are querying this route anyways to display the map. Can you just use the area data that's being provided from the map outline endpoint?